frontend/packages/dev-console/src/components/topology/components/nodes/KnativeService.scss

```diff
@@ -1,46 +1,51 @@
+@import '../../topology-utils';
+
 .odc-knative-service {
-  fill: #e2e2e2;
-  stroke: #bbbbbb;
-  stroke-width: 4px;
+  outline: none;
   cursor: pointer;
 
   .odc-m-drag-active & {
     cursor: grab;
   }
 
-  &__label {
-    cursor: pointer;
-    .odc-m-drag-active & {
-      cursor: grab;
-    }
+  &__bg {
+    fill: #d1d1d1;
+    fill-opacity: 0.5;
+    stroke: #bbbbbb;
```

Comment:
We use `#d1d1d1` and `#bbbbb` in the three groups' styles. Could we add these as variables in the `topology-utils.scss` ?